C/kadaneMaxSumSubArray.c

```diff
@@ -0,0 +1,45 @@
+#include<stdio.h>
+#define max(a,b) (a>b?a:b)
+
+int currSum = 0, maxSum = 0;
+
+int main(){
+
+	int totalElements;
+	printf("enter total size of the array\n");
+	scanf("%d", &totalElements);
+
+	int arr[totalElements];
+
+	for(int i = 0; i < totalElements; i++){
+
+		scanf("%d",&arr[i]);
+
+	}
+
+	for(int i = 0; i < totalElements; i++){
+
+		currSum += arr[i];
+
+		if(currSum < 0){
+
+			currSum = 0;
+		}
+
+		maxSum = max(maxSum, currSum);
+	}
+
+	printf("Kadane's algorithm : max sum of contigous element : %d", maxSum);
+
+}
+
```

Comment:
Please make use of 1 single multiline comment below this.

Reply:
sure!.